Xamarin.Forms.Platform.iOS/Renderers/ShellTableViewSource.cs

```diff
@@ -121,9 +122,14 @@ public override nfloat GetHeightForFooter(UITableView tableView, nint section)
 				return 1;
 			return 0;
 		}
-
 		public override UIView GetViewForFooter(UITableView tableView, nint section)
 		{
+			if (_footerCount >= Groups.Count - 2)
```

Comment:
@pictos it looks like `GenerateFlyoutGrouping` is generating an empty group at the end of the list

If we fix `GenerateFlyoutGrouping`  to not create that last empty group will that have the same effect?

Reply:
@PureWeen, it's been a while since I worked on this code, so I need to review it again to be sure.